spec/lib/pubmed/query_author_spec.rb

```diff
@@ -37,5 +37,37 @@
         expect(query_author.send(:term)).to eq('((Altman Russ[Author]) OR (Altman R[Author])) AND (Stanford University[Affiliation])')
       end
     end
+
+    context 'with a user with alternate identities with different institutions' do
+      before do
+        AuthorIdentity.create(
+          author: author,
+          first_name: 'R',
+          middle_name: 'B',
+          last_name: 'Altman',
+          institution: 'Amherst College'
+        )
+      end
+
+      it 'generates the correct term string' do
+        expect(query_author.send(:term)).to eq('((Altman Russ[Author]) OR (Altman R[Author])) AND (Stanford University[Affiliation] OR Amherst College[Affiliation])')
+      end
+    end
+
+    context 'with a user with alternate identities with different institutions with an &' do
+      before do
+        AuthorIdentity.create(
+          author: author,
+          first_name: 'R',
+          middle_name: 'B',
+          last_name: 'Altman',
+          institution: 'William & Mary'
+        )
+      end
+
+      it 'generates the correct term string' do
+        expect(query_author.send(:term)).to eq('((Altman Russ[Author]) OR (Altman R[Author])) AND (Stanford University[Affiliation] OR William Mary[Affiliation] OR William and Mary[Affiliation])')
```

Comment:
Are we confident the extra `and` isn't going to screw up the query parser?   Is there a way to designate "William and Mary" is one token rather than 3?

Reply:
Determined this approach by testing as described in #1059, which isn't to say that there are not other ways of formulating the query.

Reply:
I see we're doing `\"oregon\"[MeSH Terms]` would it be possible to do `\"William & Mary\"[Affiliation]`?

Reply:
I tried various variations with escaped quoting, escaping the ampersand, and using parentheses to no avail.

Reply:
Thanks @justinlittman